### PR TITLE
SASS-3133: Remove Employment changes

### PR DESCRIPTION
**Pull request by srimatta**
### Description
Please include a brief summary of the change you've made and the issue that it fixes.

Add a link to the relevant story in Jira
[SASS-SASS-3133](https://jira.tools.tax.service.gov.uk/browse/SASS-3133)

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [ ]  Have you checked to ensure all dependencies are up to date?
- [x]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [ ]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [ ]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the journey tests? (where applicable)
- [x]  Have you addressed warnings where appropriate?
- [ ]  Have you rebased against the current version of main?
- [x]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [ ]  Have you checked the PR Builder passes?

### CODE REVIEW

File: app/services/employment/RemoveEmploymentService.scala [hunk 1]
```diff
@@ -96,18 +93,13 @@ class RemoveEmploymentService @Inject()(deleteOrIgnoreEmploymentConnector: Delet
     nrsService.submit(user.nino, nrsPayload.toNrsPayloadModel, user.mtditid, user.trueUserAgent)
   }
 
-  private def handleConnectorCall(taxYear: Int, employmentId: String, toRemove: String,
-                                  allEmploymentData: AllEmploymentData, isLastEmployment: Boolean, user: User)
+  private def handleConnectorCall(taxYear: Int, employmentId: String, toRemove: String, user: User)
                                  (implicit hc: HeaderCarrier): Future[Either[APIErrorModel, Unit]] = {
 
     deleteOrIgnoreEmploymentConnector.deleteOrIgnoreEmployment(user.nino, taxYear,
       employmentId, toRemove)(hc.withExtraHeaders("mtditid" -> user.mtditid)).flatMap {
```
Comment (jim-collins) [inline app/services/employment/RemoveEmploymentService.scala:100]: Could this be a map and just return Left or Right?

**Review by jim-collins — APPROVED** [review]

File: app/views/employment/RemoveEmploymentView.scala.html [hunk 2]
```diff
@@ -41,20 +44,29 @@
...
+                    <span class="govuk-body-l">@{messages("common.started.date.missing")}</span>
+                }
+            </h2>
+        </div>
 
-            <p class="govuk-body">@messages(s"employment.removeEmployment.remove-paragraph${if(appConfig.studentLoansEnabled)".sl"else""}")</p>
-            <p class="govuk-body">@messages(s"employment.removeEmployment.removeExpenses")</p>
+        @if(isHmrcEmployment){
+            <p id="remove-info-id" class="govuk-body">@messages(s"employment.removeEmployment.inset${if(request.user.isAgent)".agent"else""}")</p>
         }
```
Comment (jim-collins) [inline app/views/employment/RemoveEmploymentView.scala.html:61]: This is missing:
This is information we already hold about you. If the information is incorrect, speak to the employer.

File: app/views/employment/RemoveEmploymentView.scala.html [hunk 3]
```diff
@@ -41,20 +44,29 @@
...
+        <div class="govuk-inset-text">
+            <h2  class="govuk-heading-m">
+                @{messages(employerName)} <br/>
+                @if(startDate.nonEmpty) {
+                    <span class="govuk-body-l">@{messages("employment.removeEmployment.started.date", dateFormatter(startDate).get)}</span>
+                }else {
+                    <span class="govuk-body-l">@{messages("common.started.date.missing")}</span>
+                }
+            </h2>
+        </div>
```
Comment (jim-collins) [inline app/views/employment/RemoveEmploymentView.scala.html:56]: The Period of employment styling is different to the prototype. I am not sure if it needs to change. If it doesn't I am happy to sign it off

**Review by jim-collins — CHANGES_REQUESTED** [review]

**Review by jim-collins — APPROVED** [review]

File: app/services/employment/RemoveEmploymentService.scala (hunk 1 above)
Comment (georgievh) [reply app/services/employment/RemoveEmploymentService.scala:100]: Yes, this was the suggestion that I made during the group review.

**Pull request merged**